src/App.js

```diff
@@ -14,11 +16,33 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    load: false,
```

Comment:
```suggestion
    isLoaded: false,
```
variables for boolean values should sound as if you try to answer yes-no to it